Example/Pods/PINCache/PINCache/PINCache.h

```diff
@@ -94,18 +94,47 @@ typedef void (^PINCacheObjectContainmentBlock)(BOOL containsObject);
  @param name The name of the cache.
  @result A new cache with the specified name.
  */
-- (instancetype)initWithName:(NSString *)name;
+- (instancetype)initWithName:(nonnull NSString *)name;
```

Comment:
This is assumed to be non-null because of `NS_ASSUME_NONNULL_BEGIN`, right?  https://developer.apple.com/swift/blog/?id=25

Same for a few places below.